tr_barcode/tr_barcode.py

```diff
@@ -33,6 +32,9 @@
     _logger.warning("ERROR IMPORTING REPORT LAB")
```

Comment:
Erm here is a case for which this rule is wrong unless it is possible to define logger just before `try .. except`